app/controllers/group_order_invoices_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GroupOrderInvoicesController < ApplicationController
+  include Concerns::SendGroupOrderInvoicePdf
+
+  def show
+    @group_order_invoice = GroupOrderInvoice.find(params[:id])
+    if FoodsoftConfig[:contact][:tax_number]
+      respond_to do |format|
+        format.pdf do
+          send_group_order_invoice_pdf @group_order_invoice if FoodsoftConfig[:contact][:tax_number]
+        end
+      end
+    else raise RecordInvalid
+      redirect_back fallback_location: root_path, notice: 'Something went wrong', :alert => I18n.t('errors.general_msg', :msg => "#{error} " + I18n.t('errors.check_tax_number'))
```

Comment:
* what kind of error do you want to catch here?
* please use translations for user visible strings.
* please do not use `notice` and `alert` at the same time
* there is usually no need to use `I18n.t` and you could use just `t` in the controllers
* please use newer syntax `name: value` instead of `:name => value`